scipy/linalg/_matfuncs_sqrtm.py

```diff
@@ -185,12 +185,8 @@ def sqrtm(A, disp=True, blocksize=64):
             # float byte size range: f2 ~ f16
             X = X.astype(f"f{np.clip(byte_size, 2, 16)}", copy=False)
         else:
-            # complex byte size range: c8 ~ c32.
-            # c32(complex256) might not be supported in some environments.
-            if hasattr(np, 'complex256'):
-                X = X.astype(f"c{np.clip(byte_size*2, 8, 32)}", copy=False)
-            else:
-                X = X.astype(f"c{np.clip(byte_size*2, 8, 16)}", copy=False)
+            # complex byte size range: c8 ~ c16.
+			X = X.astype(f"c{np.clip(byte_size*2, 8, 16)}", copy=False)
```

Comment:
Be sure that you have your editor set to use spaces rather than tab characters.

```suggestion
            X = X.astype(f"c{np.clip(byte_size*2, 8, 16)}", copy=False)
```